src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs

```diff
@@ -30,7 +30,7 @@ public DeltaTableTests(DeltaFixture fixture)
         /// Run the end-to-end scenario from the Delta Quickstart tutorial.
         /// </summary>
         /// <see cref="https://docs.delta.io/latest/quick-start.html"/>
-        [SkipIfSparkVersionIsLessThan(Versions.V2_4_2)]
+        [SkipIfSparkVersionIsNotInRange(Versions.V2_4_2, Versions.V3_1_1)]
```

Comment:
Disable Delta tests that have code path that creates a `org.apache.spark.sql.catalyst.expressions.Alias` object.   `Delta 0.8.0` is not compatible with `Spark 3.1.1`  https://github.com/delta-io/delta/issues/594